packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -404,16 +404,42 @@
                             <div
                                 class="oui-field__control oui-field__control_m"
                             >
-                                <input
-                                    class="oui-input"
-                                    data-ng-attr-type="{{ $ctrl.shouldDisplayPasswordInput ? 'password' : 'text' }}"
-                                    id="passwordConfirmation"
-                                    name="passwordConfirmation"
-                                    required
-                                    data-ng-model="$ctrl.newAccount.passwordConfirmation"
-                                    data-ng-model-options="{ updateOn: 'blur' }"
-                                    data-ng-change="$ctrl.onPasswordConfirmationChange()"
-                                />
+                                <div class="mb-0 oui-input-overlay">
+                                    <input
+                                        class="oui-input"
+                                        data-ng-attr-type="{{ $ctrl.shouldDisplayPasswordInput ? 'password' : 'text' }}"
```

Comment:
Question: why are you using password or text in same input?

Reply:
to allow customers to display and hide password.